lib/model.d.ts

```diff
@@ -325,7 +325,7 @@ export interface IncludeOptions {
   /**
    * Load further nested related models
    */
-  include?: Includeable[];
+  include?: Includeable[] | IncludeAll[];
```

Comment:
I made a mistake by in that comment, I meant this shouldn't be possible: `[{all: true}, {exclude: ...}]` (no multi-element arrays with `all: true`).

I think the second test is out of scope because TS uses duck typing and any object that has a superset of the properties will still be accepted. So whatever other properties you add, like `include`, I would always expect it to be accepted.